README.md

```diff
@@ -264,5 +264,29 @@ For example:
   RtspTcpPort = "8554"
 ```
 
+
+### CameraStatus REST API Endpoint
+Use the following query to determine the status of the camera.
+
+URL parameter:
+- **DeviceName**: The name of the camera
+
+```
+curl -X GET http://localhost:59882/api/v2/device/name/[DeviceName]/CameraStatus?InputIndex=0 | jq -r '"CameraStatus: " + (.event.readings[].value|tostring)'
```

Comment:
Based on the previous docs to have consistency I would suggest
```suggestion
curl -X GET http://localhost:59882/api/v2/device/name/<DeviceName>/CameraStatus?InputIndex=0 | jq .
```

Reply:
the new guides have been parsing the json response like this